data/map.txt

```diff
@@ -729,7 +730,62 @@ system Remnant
 		distance 1930
 		period 1598.78
 
-
+system Successor
+	pos -9820.108 -10035.952
+	habitable 955.249
+	belt 1535 9
+	belt 2444
+	haze star/void_scar
```

Comment:
I don't know what this is supposed to be.
The file is named "void-scar" and it's used as a star sprite, not haze.